backend/src/emails/sendEmail.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import path from 'node:path'
+
+import Email from 'email-templates'
+import { createTransport } from 'nodemailer'
+// import type Email as EmailType from '@types/email-templates'
+
+import CONFIG from '@config/index'
+import logosWebapp from '@config/logos'
+import metadata from '@config/metadata'
+import { UserDbProperties } from '@db/types/User'
+
+const hasAuthData = CONFIG.SMTP_USERNAME && CONFIG.SMTP_PASSWORD
+const hasDKIMData =
+  CONFIG.SMTP_DKIM_DOMAINNAME && CONFIG.SMTP_DKIM_KEYSELECTOR && CONFIG.SMTP_DKIM_PRIVATKEY
+
+const welcomeImageUrl = new URL(logosWebapp.LOGO_WELCOME_PATH, CONFIG.CLIENT_URI)
+const settingsUrl = new URL('/settings/notifications', CONFIG.CLIENT_URI)
+
+const defaultParams = {
+  welcomeImageUrl,
+  APPLICATION_NAME: CONFIG.APPLICATION_NAME,
+  ORGANIZATION_NAME: metadata.ORGANIZATION_NAME,
+  ORGANIZATION_URL: CONFIG.ORGANIZATION_URL,
+  supportUrl: CONFIG.SUPPORT_URL,
+  settingsUrl,
+}
+
+export const transport = createTransport({
+  host: CONFIG.SMTP_HOST,
+  port: CONFIG.SMTP_PORT,
+  ignoreTLS: CONFIG.SMTP_IGNORE_TLS,
+  secure: CONFIG.SMTP_SECURE, // true for 465, false for other ports
+  pool: true,
+  maxConnections: CONFIG.SMTP_MAX_CONNECTIONS,
+  maxMessages: CONFIG.SMTP_MAX_MESSAGES,
+  auth: hasAuthData && {
+    user: CONFIG.SMTP_USERNAME,
+    pass: CONFIG.SMTP_PASSWORD,
+  },
+  dkim: hasDKIMData && {
+    domainName: CONFIG.SMTP_DKIM_DOMAINNAME,
+    keySelector: CONFIG.SMTP_DKIM_KEYSELECTOR,
+    privateKey: CONFIG.SMTP_DKIM_PRIVATKEY,
+  },
+})
+
+const email = new Email({
+  message: {
+    from: `${CONFIG.APPLICATION_NAME}`,
+  },
+  transport,
+  i18n: {
+    locales: ['en', 'de'],
+    defaultLocale: 'en',
+    retryInDefaultLocale: false,
+    directory: path.join(__dirname, 'locales'),
+    updateFiles: false,
+    objectNotation: true,
+    mustacheConfig: {
+      tags: ['{', '}'],
+      disable: false,
+    },
+  },
+  preview: false,
+  // This is very useful to see the emails sent by the unit tests
+  /*
+  preview: {
+    open: {
+      app: 'brave-browser',
+    },
+  },
+  */
+})
+
+interface OriginalMessage {
+  to: string
+  from: string
+  attachments: string[]
+  subject: string
+  html: string
+  text: string
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const sendNotificationMail = async (notification: any): Promise<OriginalMessage> => {
```

Comment:
Why did you decide to keep this at `any`? Didnt you say it was your goal to specify this especially?